sampleReceiverApp/src/main/java/org/kinecosystem/sampleReceiverApp/ReceiveKinService.java

```diff
@@ -13,12 +13,13 @@ public ReceiveKinService() {
 
     @Override
     public void onTransactionCompleted(@NonNull String fromAddress, @NonNull String toAddress, int amount, @NonNull String transactionId, @NonNull String memo) {
-        Log.d("####", "##### developer  inform of transaction complete from " + fromAddress + " to " + toAddress+ " amount " + amount+ " transactionId " + transactionId+ " memo " + memo);
+        Log.d("ReceiveKinService", "Transaction completed from " + fromAddress + " to " + toAddress+ " amount " + amount+ " transactionId " + transactionId+ " memo " + memo);
```

Comment:
do you think we should run toast on ui that tell the receveiver got notification?

Reply:
No. As a matter a fact it could be a choice whether to implement the service/broadcast receiver that receives the notification.